airflow/models/connection.py

```diff
@@ -380,3 +381,17 @@ def extra_dejson(self) -> Dict:
                 self.log.error("Failed parsing the json for conn_id %s", self.conn_id)
 
         return obj
+
+    @classmethod
+    def get_connections_from_secrets(cls, conn_id: str) -> List['Connection']:
```

Comment:
```suggestion
    def get_connections_from_secrets(cls, conn_id: str) -> List[Connection]:
```

Reply:
Unfottunately It won't work - at this time "Connection" is not yet defined and mypy will complain (because it is inside the Connection definition - quoting is specifically foreseen for that case https://www.python.org/dev/peps/pep-0484/#forward-references

Reply:
Oh yes, you are right